records_mover/records/schema/field/__init__.py

```diff
@@ -114,6 +115,10 @@ def python_type_to_field_type(specific_type: Type[Any]) -> Optional['FieldType']
             datetime.date: 'date',
 
             datetime.time: 'time',
+
+            datetime.datetime: 'datetime',
+
+            pd.Timestamp: 'datetime',
```

Comment:
The component test I added revealed a couple of types we could handle a little better when downcasting dataframes.